scripts/generator/graphql_generator/csharp/ShopifyBuy.cs.erb

```diff
@@ -485,6 +485,60 @@ namespace Shopify.Unity {
             }, first: first, after: after);
         }
 
+        /// <summary>
+        /// Generates a query to fetch the specified <c>collections</c> by id from a Shopify store. The generated query will query the following on collections:
```

Comment:
nit: can the second sentence be on a new line? (makes it easier to read in narrow displays)